client/src/index.html

```diff
@@ -4,6 +4,9 @@
   <meta charset="utf-8">
   <title>Open Pension</title>
   <base href="/">
+  <!-- semantic-ui -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/semantic.min.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/semantic.min.js"></script>
```

Comment:
@DavidKohav `semantic-ui` scripts are dependent on `jQuery`, so pls add this too (above the `semantic-ui` `js`, but under the `css`.

Google CDN will do the work: `<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>`.